fieldservice/__manifest__.py

```diff
@@ -11,6 +11,7 @@
     'depends': [
         'base_geolocalize',
         'mail',
+        'stock',
```

Comment:
Why are you adding this dependency? We don't want to depend on stock in this module.